assets/js/base/components/payment-methods/express-payment-methods.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { getExpressPaymentMethods } from '@woocommerce/blocks-registry';
+import { useCheckoutData, usePaymentEvents } from '@woocommerce/base-hooks';
+
+const ExpressPaymentMethods = () => {
+	const [ checkoutData ] = useCheckoutData();
+	const { dispatch, select } = usePaymentEvents();
+	const paymentMethods = getExpressPaymentMethods();
+	const paymentMethodSlugs = Object.keys( paymentMethods );
+	const content =
+		paymentMethodSlugs.length > 0 ? (
+			paymentMethodSlugs.map( ( slug ) => {
+				const ExpressPaymentMethod = paymentMethods[ slug ];
+				const paymentEvents = { dispatch, select };
+				return (
+					<li
+						key={ `paymentMethod_${ slug }` }
+						id={ `express-payment-method-${ slug }` }
+					>
+						<ExpressPaymentMethod
+							checkoutData={ checkoutData }
+							paymentEvents={ paymentEvents }
+						/>
+					</li>
+				);
+			} )
+		) : (
+			<li key="noneRegistered">No registered Payment Methods</li>
```

Comment:
Yes it would.  However, this is mostly just dummy content for now. In our context we'll probably not even show this and have a placeholder pointing users to their Woo Payment Method settings page to setup a payment method (under the assumption they haven't setup any yet).